### PR TITLE
Add basic docker things

### PR DESCRIPTION
**Pull request by Lomey**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: password
+  web:
+    image: devchallenge-platform_web:latest
+    volumes:
+      - .:/devchallenge
+    ports:
+      - "8081:8080"
+    depends_on:
+      - db
```
Comment (sveredyuk) [inline docker-compose.yml:32]: + sikideq

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+COPY public ./public
+COPY storage ./storage
+COPY vendor ./vendor
+COPY config.ru .
+
+# Precompile assets
+RUN rails assets:precompile
+
+COPY run_container.sh .
+RUN chmod +x ./run_container.sh
```
Comment (PyrinAndrii) [inline Dockerfile]: I thin we should change it:
```suggestion
COPY entrypoint.sh .
RUN chmod +x ./entrypoint.sh
```

**Review by PyrinAndrii — COMMENTED** [review]
Doesn't work for me. Looks like we should add `secret_key_base` to `staging` credentials.

Screenshot 2022-04-18 at 20 45 59

File: Dockerfile (hunk 2 above)
Comment (Lomey) [reply Dockerfile]: Actually no need for this code at all. Removed

Comment (Lomey) [reply Dockerfile]: Added secret_key_base to credentials, please check it out with `rails credentials:edit --environment staging`

File: docker-compose.yml (hunk 1 above)
Comment (Lomey) [reply docker-compose.yml:32]: done

**Pull request merged**